src/Umbraco.Core/Models/EntityExtensions.cs

```diff
@@ -4,15 +4,6 @@ namespace Umbraco.Core.Models
 {
     public static class EntityExtensions
     {
-
-        /// <summary>
-        /// Determines whether the entity was just created and persisted.
-        /// </summary>
-        public static bool IsNewEntity(this IRememberBeingDirty entity)
```

Comment:
I think people in the community use this method to determine if an entity is new (just created) during an event like `ContentService.Saved`

Reply:
could be... but it was rather confusing and we ended up using it in places where we should not... so proposing we add it back if/when ppl ask for it, with a better name & implementation?